scripts/delayed.js

```diff
@@ -51,3 +51,37 @@ if (window.hlx.rum.isSelected) {
   };
   document.head.appendChild(script);
 }
+
+function loadPrivacy() {
+  function getOtDomainId() {
+    const domains = {
+      'adobe.com': '7a5eb705-95ed-4cc4-a11d-0cc5760e93db',
+      'hlx.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271',
+    };
+
+    const currentDomain = Object.keys(domains)
+      .find((domain) => window.location.host.indexOf(domain) > -1);
+
+    return `${domains[currentDomain] || domains[Object.keys(domains)[0]]}`;
+  }
+
+  // Configure Privacy
+  window.fedsConfig = {
+    privacy: {
+      otDomainId: getOtDomainId(),
+    },
+  };
+
+  const preferenceCenter = document.querySelector('[href="https://www.adobe.com/#openPrivacy"]');
+  if (preferenceCenter) {
+    preferenceCenter.addEventListener('click', (event) => {
+      event.preventDefault();
+      window?.adobePrivacy?.showConsentPopup();
+    });
+  }
+
+  const env = getHelixEnv().name === 'prod' ? '' : 'stage.';
+  loadScript(`https://www.${env}adobe.com/etc/beagle/public/globalnav/adobe-privacy/latest/privacy.min.js`);
```

Comment:
privacy.min.js loads OTSDKStub.js which loads:
* configuration for domain - e.g. https://cdn.cookielaw.org/consent/7a5eb705-95ed-4cc4-a11d-0cc5760e93db/7a5eb705-95ed-4cc4-a11d-0cc5760e93db.json
* banner file - https://cdn.cookielaw.org/scripttemplates/6.9.0/otBannerSdk.js
* network call to get location - https://geolocation.onetrust.com/cookieconsentpub/v1/geo/location
* localized content - https://cdn.cookielaw.org/consent/7a5eb705-95ed-4cc4-a11d-0cc5760e93db/8fc5213e-cec6-4fca-a134-aec9029b0675/en.json
* banner css - https://cdn.cookielaw.org/scripttemplates/6.9.0/assets/otFlat.json
* PC css - https://cdn.cookielaw.org/scripttemplates/6.9.0/assets/otPcCenter.json

The files are loaded every single time by the OneTrust script.